openlibrary/plugins/upstream/jsdef.py

```diff
@@ -63,9 +63,11 @@
     SuiteNode,
     DefNode,
     PythonTokenizer,
-    INDENT,
+    # INDENT,
 )
 
+INDENT = "    "
```

Comment:
Is this export needed upstream?

Reply:
Across the whole repo...
% `git grep INDENT`
```
openlibrary/plugins/upstream/jsdef.py:    INDENT,
openlibrary/plugins/upstream/jsdef.py:INDENT = "    "
openlibrary/plugins/upstream/jsdef.py:        text += self.jsemit(node.suite, indent + INDENT)
openlibrary/plugins/upstream/jsdef.py:        text += self.jsemit(node.suite, indent + INDENT)
openlibrary/plugins/upstream/jsdef.py:        text += self.jsemit(node.suite, indent + INDENT)
```